### PR TITLE
Wrote Configurator VTX Tab Wiki

### PR DESCRIPTION
**Pull request by YarosMallorca**

### CODE REVIEW

File: docs/wiki/configurator/vtx-tab.md [hunk 1]
```diff
@@ -4,10 +4,63 @@ sidebar_position: 15
 
 # Video Transmitter Tab
 
+Set up your video transmitter using this tab. It allows you to set the power, band, and channels for your VTX.
+
 :::note
-Sorry; this Wiki page has not yet been written.
+If you're using a DJI or Walksnail video transmitters, you can ignore this tab.
```
Comment (haslinghuis) [inline docs/wiki/configurator/vtx-tab.md:10]:
```suggestion
When using a DJI or Walksnail video transmitters, this tab can be ignored.
```

File: docs/wiki/configurator/vtx-tab.md [hunk 2]
```diff
@@ -4,10 +4,63 @@ sidebar_position: 15
...
+| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Enter frequency directly** | If you enable this, the Configurator will let you select a frequency in place of the habitual band/channel. For this to work your VTX must support this feature.                                                                                                                                                                                                                    |
```
Comment (haslinghuis) [inline docs/wiki/configurator/vtx-tab.md:22]:
```suggestion
| **Enter frequency directly** | When enabled, the Configurator will let you select a frequency in place of the habitual band/channel. For this to work your VTX must support this feature.                                                                                                                                                                                                                    |
```

**Review by haslinghuis — APPROVED** [review]

File: docs/wiki/configurator/vtx-tab.md [hunk 3]
```diff
@@ -4,10 +4,63 @@ sidebar_position: 15
 
 # Video Transmitter Tab
 
+Set up your video transmitter using this tab. It allows you to set the power, band, and channels for your VTX.
```
Comment (nerdCopter) [inline docs/wiki/configurator/vtx-tab.md:7]:
```suggestion
Configure the video transmitter (VTX) using this tab. It allows setting of the power, band, channels, etc.
```

**Comment by nerdCopter** [comment]
@YarosMallorca , please re-write without the use of `you` and `your` etc.  Generalize it overall with "the vtx", "the tab" or similar. when needed, maybe use "user" or "pilot" , but i think removal you and your will be rather easy.

**Comment by YarosMallorca** [comment]
@nerdCopter done! Please check again.

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**